src/rich-text-editor.component.js

```diff
@@ -27,6 +27,21 @@ export const RichTextEditor = forwardRef((props, editorRef) => {
     }
   }
 
+  function interceptPaste (event) {
+    // https://developer.mozilla.org/en-US/docs/Web/Events/paste
+    event.preventDefault()
```

Comment:
By preventing default, the paste won't happen, right? So don't we need to insert the HTML manually to make the paste actually work?

Reply:
correct, but the idea here is that the user wants to strip/alter paste perhaps in which case they have to setHTML... or maybe bandicoot just takes the returned value and sets HTML?

```js
function stripHTML(content) {
  ...
}
return (
<RichTextEditor 
  ref={editorRef}
  pasteFn={content => editorRef.current.setHTML(stripHTML(content))

```

Reply:
But pasting shouldn't override the entire rich text editor's content. Only the highlight portion. Right?

Reply:
Have you tested this in Edge? MDN says that it's support is unknown. I just tested it locally and it seemed to work fine, but we'll have to keep an eye on it. https://developer.mozilla.org/en-US/docs/Web/API/Range/createContextualFragment